src/aleph/handlers/content/store.py

```diff
@@ -122,9 +119,14 @@ async def fetch_related_content(
         do_standard_lookup = True
 
         # Sentinel value, the code below always sets a value but mypy does not see it.
+        # otherwise if config.storage.store_files is False, this will be the database value
         size: int = -1
 
-        if engine == ItemType.ipfs and ipfs_enabled:
+        if (
+            config.storage.store_files.value
```

Comment:
There are 2 types of file storage: IPFS and local on the node. My understanding is that `config.storage.store_files.value` is for the local storage and `config.ipfs.enabled.value -> ipfs_enabled` for IPFS storage.

Is it therefore justified to diable IPFS storage when `config.storage.store_files.value == False` and `config.ipfs.enabled.value == True` ?

Reply:
Probably not yeah, I'll remove that.